protocols/interfaces.go

```diff
@@ -15,8 +15,9 @@ type Message struct {
 
 // Transaction is an object to be sent to a blockchain provider.
 type Transaction struct {
```

Comment:
Not changed in this PR but:
`Transaction` should probably be `ChainTransaction` or `ChainTx` for disambiguation.

Reply:
#157